src/fetch/period_slippage.py

```diff
@@ -2,17 +2,20 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+from enum import Enum
 from pprint import pprint
 
 from src.dune_analytics import DuneAnalytics, QueryParameter
-from src.file_io import File
 from src.models import AccountingPeriod, Address, Network
-from src.token_list import ALLOWED_TOKEN_LIST_URL, get_trusted_tokens_from_url
+from src.token_list import fetch_trusted_tokens
```

Comment:
The url is a constant and can be entirely contained within the method that uses it.